packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsCompletedRow/TransactionsCompletedRow.tsx

```diff
@@ -22,37 +23,40 @@ const TransactionsCompletedRow: React.FC<TProps> = ({ accounts, transaction, wal
             : transaction.action_type.replace(/^\w/, c => c.toUpperCase());
 
     return (
-        <div className='wallets-transactions-completed-row'>
-            {transaction.action_type !== 'transfer' ? (
-                <TransactionsCompletedRowAccountDetails
-                    accountType={wallet?.account_type ?? ''}
-                    actionType={transaction.action_type}
-                    currency={wallet?.currency ?? 'USD'}
-                    displayAccountName={displayWalletName}
-                    displayActionType={displayActionType}
-                    isDemo={Boolean(wallet?.is_virtual)}
-                />
-            ) : (
-                <TransactionsCompletedRowTransferAccountDetails
-                    accounts={accounts}
-                    direction={transaction.from?.loginid === wallet?.loginid ? 'to' : 'from'}
-                    loginid={
-                        [transaction.from?.loginid, transaction.to?.loginid].find(
-                            loginid => loginid !== wallet?.loginid
-                        ) ?? ''
-                    }
-                />
-            )}
-            <div className='wallets-transactions-completed-row__transaction-details'>
-                <WalletText color={transaction.amount > 0 ? 'success' : 'error'} size='xs' weight='bold'>
-                    {transaction.amount && transaction.amount > 0 ? '+' : ''}
-                    {transaction.display_amount}
-                </WalletText>
-                <WalletText color='primary' size='2xs'>
-                    Balance: {transaction.display_balance_after}
-                </WalletText>
+        <React.Fragment>
+            <Divider color='#e5e5e5' />
```

Comment:
Do we not have variable for this colour?

Reply:
There's no variable for this colour but got confirmation with designers and updated to var(--border-divider) or #f2f3f4 instead